channels/OFF-CHAIN.md

```diff
@@ -0,0 +1,503 @@
+# Off-chain
+
+## Messages
+
+- [Overview](#overview)
+- [Control messages](#control-messages)
+	- [`error`](#error)
+	- [`ping`/`pong`](#ping_pong)
```

Comment:
Empty section and invalid anchor (`ping_pong` while working one appears to be `#pingpong`). Maybe section shall be deleted?

Reply:
I am fixing the broken link; WRT empty section - I think it shall be filled